### PR TITLE
Mark play_end_to_end test as xfail in Windows

### PR DESCRIPTION
**Pull request by Crola1702**
As mentioned in https://github.com/ros2/rosbag2/issues/1437

### CODE REVIEW

**Comment by Crola1702** [comment]
CI: 
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19380/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13904/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/20100/)

**Comment by emersonknapp** [comment]
Should we only do this on windows?

File: rosbag2_tests/CMakeLists.txt
Comment (MichaelOrlov) [inline rosbag2_tests/CMakeLists.txt]:
```suggestion
if(WIN32)
    ament_add_test_label(test_rosbag2_play_end_to_end xfail)
endif()
```

**Review by MichaelOrlov — CHANGES_REQUESTED** [review]
I think it makes sense to mark the test as xfail only for the Windows platform seems it only regularly fails on Windows.

File: rosbag2_tests/CMakeLists.txt
Comment (Crola1702) [reply rosbag2_tests/CMakeLists.txt]: Yes you're right. I totally missed that.

**Comment by Crola1702** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19399/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13928/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/20120/)

**Review by clalancette — APPROVED** [review]
Looks reasonable to me, but I'd like to wait for final confirmation from @MichaelOrlov before merging.

**Review by MichaelOrlov — APPROVED** [review]
:+1:

**Comment by MichaelOrlov** [comment]
@clalancette Sorry, I haven't had a chance yet to make a comprehensive analysis about the reason why this test fails on Windows. But as I said before, I don't mind disabling this test for Windows for a while. Anyway, I will have to reproduce the failure locally.

**Comment by clalancette** [comment]
> @clalancette Sorry, I haven't had a chance yet to make a comprehensive analysis about the reason why this test fails on Windows. But as I said before, I don't mind disabling this test for Windows for a while. Anyway, I will have to reproduce the failure locally.

No worries at all, thanks for any time you spend on it.  I think disabling it just for Windows is a good compromise for now.

**Pull request merged**
